Websites/perf.webkit.org/tools/js/buildbot-syncer.js

```diff
@@ -148,7 +148,7 @@ class BuildbotSyncer {
         return this._remote.postJSON(path, data);
     }
 
-    scheduleRequestInGroupIfAvailable(newRequest, requestsInGroup, workerName)
+    scheduleRequestInGroupIfAvailable(newRequest, requestsInGroup, workerName, useIdleWorker=false)
```

Comment:
useIdleWorker sounds rather generic.
We probably want to use a more specific term like useWorkerWithoutRequestsInProgress.